eZ/Bundle/EzPublishIOBundle/BinaryStreamResponse.php

```diff
@@ -133,6 +137,11 @@ public function setContentDisposition( $disposition, $filename = '', $filenameFa
      */
     public function prepare( Request $request )
     {
+        if ( !$this->file instanceof BinaryFile )
```

Comment:
cool, witth that change I think we are ready for merge, could you create a issue (or there might be one already?) and rebase this branch to have one commit including reference to issue in commit massage, and the change agreed to here?